tests/integration-tests/tests/performance_tests/test_starccm/test_starccm/starccm.run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
This file is not needed anymore since you're submitting the job with remote command executor